### PR TITLE
Changelog: odahuflowctl local pack run changes info

### PR DESCRIPTION
**Pull request by vlad-tokarev**

### CODE REVIEW

**Comment by keshamin** [comment]
@vlad-tokarev Do you want to merge all changes at once in this single PR when it's time to release 1.4 or to merge the changes incrementally?

File: source/ref_changelog.rst [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+--------------------------
+
+Features:
+""""""""""""
+
+Updates:
+""""""""""""
+
+- CLI:
+    * We added ``odahuflowctl local pack run`` command's new option ``--disable-target`` that allows you disable targets
```
Comment (BPylypenko) [inline source/ref_changelog.rst]:
```suggestion
    * We added ``--disable-target`` option for ``odahuflowctl local pack run`` command that allows you disable targets
```
Maybe changes stress a little but, for me, it's easier to read.

File: source/ref_changelog.rst [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+      which will be passed to packager process. You can use multiple options at once. For example:
+      ``odahuflowctl local pack run ... --disable-target=docker-pull --disable-target=docker-push``.
+    * ``odahuflowctl local pack run`` command's options ``--disable-package-targets/--no-disable-package-targets`` are
+      deprecated.
+    * ``odahuflowctl local pack run`` behavior that implicitly disable all targets by default is deprecated.
```
Comment (BPylypenko) [inline source/ref_changelog.rst]:
```suggestion
    * ``odahuflowctl local pack run`` behavior that implicitly disables all targets by default is deprecated.
```

**Review by BPylypenko — CHANGES_REQUESTED** [review]

**Comment by vlad-tokarev** [comment]
> @vlad-tokarev Do you want to merge all changes at once in this single PR when it's time to release 1.4 or to merge the changes incrementally?

I suggest merging the changes incrementally as soon as the related feature is merged

**Review by BPylypenko — APPROVED** [review]

**Review by keshamin — APPROVED** [review]

**Review by easokol — APPROVED** [review]

**Review by karbyshevds — APPROVED** [review]

**Pull request merged**